src/components/DatePicker/DatePicker.test.tsx

```diff
@@ -596,20 +599,21 @@ describe("DatePicker", () => {
       // Helper text for the "To" input
       expect(screen.getByText(/Note for the 'to' field./i)).toBeInTheDocument();
 
-      const fromInput = screen.getByRole("textbox", { name: "From" });
-      const toInput = screen.getByRole("textbox", { name: "To" });
+      const inputs = screen.getAllByRole("textbox", {
+        name: "Press tab to access the calendar.",
```

Comment:
You mentioned this before about the order of precedence that RTL uses to get by role, can you include a comment that references that?

Reply:
I added a comment on this in utils/utils.ts. Let me know if you think that is sufficient.